diff_cover/diff_cover_tool.py

```diff
@@ -49,19 +52,19 @@ def parse_coverage_args(argv):
     valid options:
 
         {
-            'coverage_xml': COVERAGE_XML,
+            'coverage_file': COVERAGE_FILE,
             'html_report': None | HTML_REPORT,
             'json_report': None | JSON_REPORT,
             'external_css_file': None | CSS_FILE,
         }
 
-    where `COVERAGE_XML`, `HTML_REPORT`, `JSON_REPORT`, and `CSS_FILE` are paths.
+    where `COVERAGE_FILE`, `HTML_REPORT`, `JSON_REPORT`, and `CSS_FILE` are paths.
 
     The path strings may or may not exist.
     """
     parser = argparse.ArgumentParser(description=DESCRIPTION)
 
-    parser.add_argument("coverage_xml", type=str, help=COVERAGE_XML_HELP, nargs="+")
+    parser.add_argument("coverage_file", type=str, help=COVERAGE_FILE_HELP, nargs="+")
```

Comment:
I'd prefer some attempt to preserve backwards compatibility.


Perhaps including the coverage xml file as an alternative argument but that does the same thing (if both are provided I'm fine with preferring `coverage_file`. Or maybe throw an error if both are provided?)

Renaming the arg everywhere else makes sense. I just would like the command line to support coverage_xml as an alternate form

Reply:
Thanks for the feedback, I'll take care of it.

Reply:
btw, can you please enable the workflows for this PR?

Reply:
I got a bit confused. This line defines a positional argument (see [ArgumentParser.add_argument](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_argument)), therefore nobody will call it with the argument name; it will only define the key in the resulting structure, which is only used internally.

Looking at [test_diff_cover_tool.py#L9](https://github.com/Bachmann1234/diff_cover/blob/main/tests/test_diff_cover_tool.py#L9) shows that nothing changed about how to call the function, therefore there's no backward compatibility issue. 

Can you please double-check if my understanding is correct?

Reply:
Yeah, I think you are right. Sorry. Misread the argparse code :-)